app/services/SessionService.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// /*
+//  * Copyright 2021 HM Revenue & Customs
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the "License");
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an "AS IS" BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+//  */
+
+// package services
+
+// import config.AppConfig
+// import javax.inject.{Inject, Singleton}
+// import models.{GuidanceSession, GET, POST}
+// import play.api.Logger
+// import core.models.errors.{InternalServerError, InvalidProcessError, AuthenticationError}
+// import core.models.RequestOutcome
+// import uk.gov.hmrc.http.HeaderCarrier
+// import play.api.i18n.{Lang, MessagesApi}
+// import scala.concurrent.{ExecutionContext, Future}
+// import repositories.SessionRepository
+// import core.models.ocelot.{LabelCache, Labels, Process}
+
+// @Singleton
+// class SessionService @Inject() (
+//     appConfig: AppConfig,
+//     sessionRepository: SessionRepository,
+//     messagesApi: MessagesApi
+// ) {
+
```

Comment:
Any value in this file if everything is commented out?

Reply:
This file is deleted

Reply:
Confirmed - resolved